FreeTAKServer/controllers/DatabaseControllers/ActiveEmergencysController.py

```diff
@@ -6,18 +6,19 @@
 class ActiveEmergencysController(TableController):
     def __init__(self):
         self.table = ActiveEmergencys
-    
+
     def create(self, session, object):
         row = ActiveEmergencys()
         row.event = EventTableController().convert_model_to_row(object, row.event)
         session.add(row)
         session.commit()
         return row
 
+
 if __name__ == "__main__":
-    from FreeTAKServer.controllers.DatabaseControllers.DatabaseController import DatabaseController
```

Comment:
This should likely stay for the commented out code, but that might be delegated to `SimulateSentCOT()`?